packages/test/package.json

```
@@ -3,24 +3,46 @@
   "version": "3.0.1",
   "description": "Testing utilities for Rest Hooks",
   "sideEffects": false,
-  "main": "dist/index.cjs.js",
-  "module": "lib/index.js",
+  "main": "dist/index.cjs",
+  "module": "legacy/index.js",
   "types": "lib/index.d.ts",
+  "exports": {
+    ".": [
+      {
+        "node": {
+          "import": "./node.mjs",
+          "require": "./dist/index.cjs",
+          "types": "./lib/index.d.ts"
+        },
+        "types": "./lib/browser.d.ts",
+        "default": "./lib/browser.js"
+      },
+      "./lib/browser.js"
+    ],
+    "./package.json": "./package.json"
+  },
+  "type": "module",
+  "engines": {
+    "node": ">=10 <13 || ^13.7 || >=14"
```

if you want to restrict to only node versions that use "exports", you'd want:
```suggestion
    "node": "^12.17 || ^13.7 || >=14"
```

---

I still have "main" so I intend to support v10

---

do you intend to have other entry points besides the package itself?

---

does 12.17 do the new behavior and not the 13.1 behavior?

---

yes, v12.17 has the full "exports" field behavior. i'm asking because the choice to use `.cjs` will cause problems if you want to support node older than v12.17, for non-main entrypoints.

---

Not currently - but I'll keep that in mind with other package designs. Thanks for all the knowledge about node version history!